rpc/backend/evm_backend.go

```diff
@@ -871,7 +871,7 @@ func (b *Backend) BaseFee(blockRes *tmrpctypes.ResultBlockResults) (*big.Int, er
 	}
 
 	if res.BaseFee == nil {
```

Comment:
I dont think this assumption is accurate. There could be other cases where the `BaseFee` is nill

Reply:
Also, might be good to improve the error message

Reply:
Yeah, it could be london hardfork not enabled.